internal/cluster/distribution/eks/service.go

```diff
@@ -93,6 +93,7 @@ type NodePool struct {
 	InstanceType string            `mapstructure:"instanceType"`
 	Image        string            `mapstructure:"image"`
 	SpotPrice    string            `mapstructure:"spotPrice"`
+	SubnetID     string            `mapstructure:"subnetID"`
```

Comment:
Is it `subnetID` or `subnetId`?

Reply:
It's the latter in the specification so I'm going to fix this instance to that casing as well.

Reply:
[Fixed](https://github.com/banzaicloud/pipeline/compare/3d24e483b4d790c4a89cca75cd8dc19e7c0d28c8..71229a645dddb7b72f60722526d098fa513538cb).